modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/paths/APIBuilder.path

```diff
@@ -100,6 +100,11 @@
 	<td>//div[@class='dropdown-menu show']//button[contains(text(),'Edit')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ENDPOINT_DESCRIPTION</td>
+	<td>//textarea[contains (@id,'endpointDescriptionField')]</td>
+	<td></td>
```

Comment:
Could you find out if we have some existing paths that can use?

Reply:
Done! Thanks for review it @Hijky